crystal-map-connector/src/main/kotlin/ch/ergon/dope/extension/type/arithmetic/NumberInfix.kt

```diff
@@ -21,6 +22,9 @@ fun Number.add(number: CMField<out Number>) = toDopeType().add(number.toDopeType
 
 fun CMField<out Number>.add(number: Number) = toDopeType().add(number.toDopeType())
 
+fun <KotlinType : Any, MapType : Number> KotlinType.add(other: CMConverterField<KotlinType, MapType>) =
+    other.typeConverter.write(this)!!.toDopeType().add(other.toDopeType())
```

Comment:
can't you hide this `typeConverter.write...` inside a toDopeType method?

Reply:
done